src/Displaytemplatestandard.h

```diff
@@ -64,16 +67,7 @@ void printScreen()
 
         //draw setPoint line
         u8g2.drawLine(18, 58 - (setPoint / 2), 23, 58 - (setPoint / 2));
-
-        // PID values over heat bar, TOF if active
-    #if TOF == 1
-        if (percentage < 10.00) {
-            if (isrCounter < 500) {
-                u8g2.setCursor(40, 48);
-                u8g2.print(langstring_waterempty);
-            }
-        }
-    #else
+// PID values over heat bar
```

Comment:
```suggestion
    // PID values over heat bar
```

Reply:
sorry ;)

Reply:
We should think about cleaning the comments and indentation in one go :)